examples/csvpb/main.go

```diff
@@ -0,0 +1,80 @@
+package main
+
+import (
+	"context"
+	"encoding/csv"
+	"log"
+	"net/http"
+	"os"
+	"time"
+
+	"github.com/alpstable/csvpb"
+	"github.com/alpstable/gidari"
+	"golang.org/x/time/rate"
+	"google.golang.org/protobuf/types/known/structpb"
+)
+
+type exampleWriter struct {
+	lists []*structpb.ListValue
+}
+
+func (w *exampleWriter) Write(cxt context.Context, list *structpb.ListValue) error {
+	w.lists = append(w.lists, list)
+
+	return nil
+}
+
+/**
+	Using the Gidari and CSVPB library to write HTTP response data to stdout as CSV
+**/
+
+func main() {
+	ctx := context.Background()
+
+	const api = "https://anapioficeandfire.com/api"
+
+	// First we create a service that can be used to make bulk HTTP
+	// requests to the API.
+	svc, err := gidari.NewService(ctx)
+	if err != nil {
+		log.Fatalf("failed to create service: %v", err)
+	}
+
+	w := &exampleWriter{}
```

Comment:
We don't have to use an example writer here. Do this part here:

```go
	// Create a writer object with 'encoding/csv' package,
 	// then use csvpb library to create list writer.
 	writer := csv.NewWriter(os.Stdout)
 	listWriter := csvpb.NewListWriter(writer, csvpb.WithAlphabetizeHeaders())
 ```
 
 And then pass the "listWriter" into the requests.